runtime-sdk/modules/evm/Cargo.toml

```diff
@@ -31,3 +31,6 @@ fixed-hash = "0.7.0"
 primitive-types = { version = "0.10.1", default-features = false, features = ["rlp", "num-traits"] }
 rlp = "0.5.1"
 uint = "0.9.1"
+
+[features]
+confidential = [] # Use confidential storage by default.
```

Comment:
It's not required to be a compile flag. It was a config option originally, and even just yesterday I considered switching back to that, but
* if it looks too configurable, people might try to switch from non-confidential to confidential or vice versa
* I wanted to keep the Emerald codepath almost completely unmodified since Sapphire is a bolt-on addition
* it's a bit ugly getting the module configuration at the point where the evm is calling the  storage methods

What's your thinking in favor of module config? The main one for me is easier testing with the current CI, but I'm not about to have github actions tie my hands.

Reply:
Yes I was mostly thinking about making testing (in general) easier as it is more flexible if you don't need a special build.